.gitignore

```diff
@@ -2,3 +2,4 @@
 _tmp/
 .gows.user.yml
 gows.yml
+.DS_Store
```

Comment:
In the infra repo I was advised to use only local `.gitignore` for environment/system specific files and don't add them to the repo. Not sure if it's a common practice in the step repos but I kind of like it:
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Steps tend to have a .gitignore file but I will remove `.DS_Store` from it.